.github/workflows/plugin-upgrade-test.yml

```diff
@@ -1,6 +1,7 @@
 name: Upgrade Elementor test
 
 on:
+  pull_request:
```

Comment:
Do we need a specific GH Action for this flow? Its very similar to the `playwright.yml` action

Reply:
The flow is different. We build the env with no plugin. Then we install latest (or any other version.). After that we build the plugin from our branch and do the upgrade and verify with playwright smoke test

Reply:
The default playwright workflow just installs built plugin